src/ninja.cc

```diff
@@ -1294,6 +1296,8 @@ NORETURN void real_main(int argc, char** argv) {
         kDepfileDistinctTargetLinesActionError;
   }
 
+  Status* status = new StatusPrinter(config);
```

Comment:
Hrmm, in C++11, this would be better as `std::make_unique` or `std::make_shared`

But we're using C++98 / C++03, and I don't really have a better suggestion than doing this for now